apps/src/code-studio/components/progress/ScriptOverview.jsx

```diff
@@ -89,6 +90,7 @@ const ScriptOverview = React.createClass({
       <div>
         {onOverviewPage && (
           <div>
+            <SignInOrAgeDialog/>
```

Comment:
TODO: brent - for signed in users, this currently shows the dialog and then hides it after we get async sync in info